winpr/libwinpr/sspi/Kerberos/kerberos.c

```diff
@@ -216,6 +223,20 @@ static BOOL isValidIP(const char* ipAddress)
 	return isValidIPv4(ipAddress) || isValidIPv6(ipAddress);
 }
 
+static int build_krbtgt(krb5_context ctx, krb5_data* realm, krb5_principal target)
+{
+	/* "krbtgt/" + realm + "@" + realm */
+	int len = 7 + realm->length + 1 + realm->length + 1;
```

Comment:
should be `size_t`, constants suffixed by `ULL` to force the data type to be 64bit